torch/csrc/jit/runtime/static/impl.h

```diff
@@ -32,12 +33,11 @@ class TORCH_API StaticRuntime {
 #endif
 
  private:
-  torch::jit::Module module_;
   std::shared_ptr<torch::jit::Graph> graph_;
 
   // Static runtime states
   // Value table (including weights)
-  ConstantMap workspace_;
+  mutable ConstantMap workspace_;
```

Comment:
hm... is it because you use operator[]? maybe use at() then? and as I mentioned elsewhere, using a map is pretty inefficient, we should just remap everything into indices.

Reply:
Is there any documentation on how much this affect the overall latency? Replacing it with mapping to indices would certainly be net positive. I'm just wondering in terms of framework overhead, what kind of percentage this contribute.